### PR TITLE
Proposal: terminology improvements

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

This PR is a response to an investigation into the terms used throughout Juicebox: https://juicebox.notion.site/Consolidate-Juicebox-lexicon-24aaa2b1936a4416a213ddb36dca67e4

I identified a few terms that seemed ambiguous, or that could be consolidated. This mostly related to `Distributions`, `Payouts`, `Distribute`, `Withdrawal` and `Revenue`.

Some of these changes came out of a discussion between @JayZeeDesign and me.

## Proposed changes

The following is a summary of the key changes (not exhaustive, see screenshots below for exact changes):

### ⚠️ Major changes 

| Change | Motivation |
| --- | --- |
|  Renaming `Distribution` to `Funding distribution` on the project create page and project page | "Distribution" can refer to the distribution of funds or the distribution of reserved tokens. This is an attempt to make them clearly distinct. |
| Rename `Distribute` button to `Distribute JBX` and `Distribute funds` respectively. | Related to the above |
| Move `Distribute JBX` button to the top of the Reserved JBX card on the project page| To be consistent with the funding distribution card. |
| Replace references to `revenue` with `funds` on the homepage | We don't use the word `revenue` anywhere in the app. | 
### Minor changes

**Create project page**:
- Rename `Funding` to `Funding cycle` on the project page
- Alter copy on `Funding distribution` section of project page

**Project page**
- Change `Edit token receivers` to `Edit token allocation` to be consistent with project page

## Screenshots or screen recordings

| before | after |
| --- | --- |
| before | after_create |
| Screen Shot 2022-01-15 at 5 11 00 pm | Screen Shot 2022-01-15 at 5 00 23 pm |
| Screen Shot 2022-01-15 at 5 11 10 pm | Screen Shot 2022-01-15 at 5 00 33 pm |
| Screen Shot 2022-01-15 at 5 11 22 pm | Screen Shot 2022-01-15 at 5 00 46 pm |
| Screen Shot 2022-01-15 at 5 11 42 pm | Screen Shot 2022-01-15 at 5 01 37 pm |
| Screen Shot 2022-01-15 at 5 11 52 pm| Screen Shot 2022-01-15 at 5 01 50 pm |
| Screen Shot 2022-01-15 at 5 12 19 pm| Screen Shot 2022-01-15 at 5 03 41 pm
| Screen Shot 2022-01-15 at 5 12 30 pm | Screen Shot 2022-01-15 at 5 04 21 pm |
| Screen Shot 2022-01-15 at 5 12 34 pm | Screen Shot 2022-01-15 at 5 04 27 pm |
| Screen Shot 2022-01-15 at 5 12 37 pm | Screen Shot 2022-01-15 at 5 04 37 pm |

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../../CONTRIBUTING.md#browser-support).

### CODE REVIEW

File: src/components/Dashboard/ReservedTokens.tsx [hunk 1]
```diff
@@ -143,6 +137,14 @@ export default function ReservedTokens({
           />
         </div>
       )}
+
+      {metadata?.reservedRate ? (
```
Comment (aeolianeth) [inline src/components/Dashboard/ReservedTokens.tsx:141]: This is moving the reserved tokens list _below_ the `Distribute tokens` button to be consistent with the `Funding distribution` list.

**Comment by aeolianeth** [comment]
Hey @JayZeeDesign @peripheralist, given the feedback from a few days ago in Discord, I've reverted the `WITHDRAW`->`PAYOUTS` tab naming change.

Could you both take another look at this PR please?

**Comment by peripheralist** [comment]
hey @aeolianeth , sorry to leave this hanging. i'm on board with everything here!

let's just get conflicts with `main` solved and we'll merge

**Comment by aeolianeth** [comment]
Thanks @peripheralist, conflicts resolved 🚀

**Review by peripheralist — APPROVED** [review]

**Pull request merged**